fieldpath/path_test.go

```diff
@@ -22,6 +22,13 @@ import (
 	"sigs.k8s.io/structured-merge-diff/value"
 )
 
+var (
+	_V = func(i interface{}) *value.Value {
```

Comment:
Why is this a variable and not just a function declaration?

Reply:
Yeah, these are residues from when I've made the value as a strict `interface{}`